### ISSUE TITLE
[Proposal] Hide bulk actions on XS

### ISSUE THREAD

**Issue by tabacitu** [Laravel-Backpack/ideas#70]
Since unlikely that the small mobile screen will show both the bulk buttons and a few relevant columns, I propose that on XS we:
- hide the checkbox column
- hide the bulk buttons

This should:
- make the mobile view more usable in the 90% scenario (just want to see something)
- be in line with what we do with the export buttons (we hide them on XS)

**Comment by tabacitu** [comment]
Maybe (maybe!!) it'd make sense to add a notice on XS, something like "_Some features might have been hidden due to the small screen size. Turn your device around to see everything - a lot more is visible when in landscape mode._" 

But personally I think that's not really needed.

**Comment by tabacitu** [comment]
Should fix https://github.com/Laravel-Backpack/CRUD/issues/3139

**Comment by promatik** [comment]
@tabacitu, we had this; https://github.com/Laravel-Backpack/CRUD/pull/3320.

That, allow us to do something like this;

```php
'defaultButtonTextClass' => [
    'top' => '',
    'line' => 'd-none',
    'bottom' => 'd-sm-none d-md-none d-lg-inline',
];
```

We can make new defaults on v4.2.

**Comment by tabacitu** [comment]
I agree, but this is a COULD and non-breaking. We have more important things to do in 4.2, labelled MUST. 
I'm trying very hard to not talk about anything else 😀
